README.md

```diff
@@ -33,22 +33,48 @@ PHAL:
   ovos-PHAL-plugin-homeassistant:
     host: http://<HA_IP_OR_HOSTNAME>:8123
     api_key: <HA_LONG_LIVED_TOKEN>
+    disable_intents: False # Set this to True if you want to disable the skill's primary intents if Home Assistant is not connected
```

Comment:
This specifies configuring the plugin, but code changes and tests reference the same key in skill settings; is this intentional?